packages/storybook/stories/va-table.stories.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { generateEventsDescription } from './events';
+import {
+  getWebComponentDocs,
+  componentStructure,
+  propStructure,
+} from './wc-helpers';
+
+const accordionDocs = getWebComponentDocs('va-table');
+
+export default {
+  title: 'Components/va-table',
+};
+
+const Template = args => {
+  const { caption } = args;
+  const data = [
+    [
+      'Declaration of Independence',
+      'Statement adopted by the Continental Congress declaring independence from the British Empire',
+      '1776',
+    ],
+    [
+      'Bill of Rights',
+      'The first ten ammendements of the U.S. Constitution guaranteeing rights and freedoms',
+      '1791',
+    ],
+    [
+      'Declaration of Sentiments',
+      'A document written during the Seneca Falls Convention outlining the rights that American women should be entitled to as citizens.',
+      '1848',
+    ],
+    [
+      'Emancipation Proclamation',
+      'An executive order granting freedom to slaves in designated southern states.',
+      '1863',
+    ],
+  ];
+
+  const columns = ['Document title', 'Description', 'Year'];
+  return (
+    <main>
+      <va-table caption={caption}>
+        <va-table-row slot="headers">
+          {columns.map(col => (
+            <span>{col}</span>
+          ))}
+        </va-table-row>
+
+        {data.map(row => (
+          <va-table-row>
+            {row.map(item => (
+              <span>{item}</span>
+            ))}
+          </va-table-row>
+        ))}
```

Comment:
I'm going to see if I can write a function that will be exported along with the rest of `component-library` that will accept a `fields` and `data` arg structured the same way as the current `<Table>` props. This should help with migration since we should be able to take this:

```jsx
import Table from '@department-of-veterans-affairs/component-library/Table';

// ...

const content = <Table data={data} fields={fields} />;
```
and have our ESLint rule suggest this:


```jsx
import { tableChildren } from '@department-of-veterans-affairs/component-library';

// ...

const content = <va-table>{tableChildren(fields, data)}</va-table>;
```

If it works I'll use it here in the story.